CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ project(blmc_controllers)
 if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wl,--no-as-needed")
 endif()
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -W -Wextra")
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -W -Wextra -fPIC")
```

Comment:
this can be removed all together, I do not think it buys you anything...

Reply:
This I took from the other header as well.